src/app/shared/task-provider/task-provider.component.ts

```diff
@@ -44,6 +45,7 @@ export class TaskProviderComponent implements OnInit {
     }
 
     onProviderChange(selectedProvider) {
+        this.getTimeProviders.emit();
```

Comment:
We are not emitting any value here, but the type declaration mentions a string value.

Reply:
done.